projects/packages/assets/src/class-assets.php

```diff
@@ -651,7 +651,7 @@ private static function init_domain_map_hooks( $domain, $firstcall ) {
 	 */
 	public static function filter_gettext( $translation, $text, $domain ) {
 		if ( $translation === $text ) {
-			// phpcs:ignore WordPress.WP.I18n -- we're including text domains from the most recent dependency package versions, of which there are generally a fair number, to ensure we have the most recent translations.
+			// phpcs:ignore WordPress.WP.I18n -- This is a filter hook to map the text domains from our Composer packages to the domain for a containing plugin. See p2gHKz-oRh-p2#problem-6-text-domains-in-composer-packages
```

Comment:
We should probably change these to the actual wp.me link rather than our shorthand. The target blog post is public.

Or we could do the full https://developer.wordpress.com/2022/01/06/wordpress-plugin-i18n-webpack-and-composer/#problem-6-text-domains-in-composer-packages URL, that just gets a little long for a comment.

Reply:
Fixed now, thanks for spotting that.